src/gui_common/tooltip/ICustomToolTip.cs

```diff
@@ -16,57 +16,56 @@ public enum ToolTipPositioning
     FollowMousePosition,
 }
 
+/// <summary>
+///   How a tooltip should transition on display and hide.
+/// </summary>
+public enum ToolTipTransitioning
+{
+    /// <summary>
+    ///   Immediately display and hide the tooltip without animation.
+    /// </summary>
+    Immediate,
+
+    /// <summary>
+    ///   Use fading to display and hide the tooltip.
+    /// </summary>
+    Fade,
```

Comment:
Actually, while playtesting I haven't found a tooltip which fades either in this PR or in the master. Did I miss some? Aren't there any? Is this broken?

Reply:
There currently aren't any.